### DISCUSSION TITLE
'Jump to bottom' button in an issue

### DISCUSSION THREAD

**Discussion by melanchall** [community/community#4565] [Mobile] [answered]
Sometimes issues get longer and it's pretty annoying to scroll them down to see latest responses. Can you please add the button to quickly jump to the bottom of an issue?

**Comment by brianlovin** [comment, marked as answer]
Thanks for the suggestion @melanchall – this isn't on our list of features to implement, but one thing we have been rolling out over the last week is better behavior for deep links and notifications that will help you get to the newest activity in the timeline more easily. Have you noticed this new behavior in the past few days?
